app/(pages)/content-collection/_component/CollectionCard/CollectionCard.module.scss

```diff
@@ -0,0 +1,122 @@
+@use 'media';
+
+.media_container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: var(--card-width); 
+    padding: var(--spacer);
+    background-color: var(--background-secondary);
+    border-radius: var(--c-radius);
+    box-shadow: var(--b-shadow);
+    gap: var(--small-spacer);
+
+    .edit_container{
+        display: flex;
+        flex-direction: column;
+        gap: var(--small-spacer);
+        > p {
+            width: 100%;
+            font-size: 1rem;
+            color: var(--text-dark-gray);
+            font-family: var(--font-dm-sans);
+        }
+    }
+
+    .title_container{
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+        > .select{
+            cursor: pointer;
+        }
+    }
+
+    .img_container{
+        width: 100%;
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        min-height: calc(9 / 16 * var(--card-width));
+        .two_img_container{
+            width: 49%;
+            position: relative; 
+            .second_img_radius{
+                border-radius: 0px var(--b-radius) var(--b-radius) 0px;
+            }
+        }
+        .first_prev_container{
+            width: 56%;
+            position: relative; 
+        }
+        .first_img_radius{
+            border-radius: var(--b-radius) 0px 0px var(--b-radius);
+        }
+    }
+
+    .media_radius{
+        border-radius: var(--b-radius);
+    }
+  
+    .title {
+      width: 100%;
+        > p {
+            color: var(--text-dark);
+            width: 100%;
+            margin: 0;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            font-size: 2rem;
+        }
+    }
+
+    .description{
+        width: 100%;
+        > p {
+            width: 100%;
+            margin: 0;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            color: var(--text-dark);
+            font-family: var(--font-dm-sans);
+        }
+    }
+
+    .preview_container{
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        width: 42%;
+        .img_prev_container{
+            height: 49%;
```

Comment:
Im not too pressed about this now, but feel free to use the calc function. I am a firm believer that spacers should be static and only should change through media queries rather than just scaling by percentages. You could declare a gap of var(--small-spacer) and then say height: calc(50% - var(--small-spacer)).